AlgebraInLean/World0LeanIntroSOLUTIONS/Sheet4Solutions.lean

```diff
@@ -0,0 +1,200 @@
+/-
+This is a solutions sheet.
+-/
+
+import Mathlib.Tactic
+
+/-Credit for some exercises goes to Kevin Buzzard and his Formalizing Mathematics
+course OR to the Natural Number Game. If you would like to learn more about Lean,
+Buzzard's course goes more in depth in relation to numerous undergraduate math topics.
+When exercises are from either of these sources, they will be clearly marked so as
+to give credit.-/
+
+/-The tactic constructor should be completely new to you. It is very useful for
```

Comment:
you might want to indicate that a tactic, such as constructor, is a tactic by changing the font or using `` before and after the word signifying the tactic.